_core/smart/cli-wallet-quickstart.md

```diff
@@ -15,7 +15,7 @@ You will need to have `npm` installed.
 
 To install the Blockstack CLI, run the following command in terminal. 
 
-`npm install -g blockstack-cli` 
+`npm install -g https://github.com/blockstack/cli-blockstack#feature/stacks-2.0-tx` 
```

Comment:
Should be just `npm install -g https://github.com/blockstack/cli-blockstack` without the anchor